tests/ostree_packman_wrapper.cc

```diff
@@ -0,0 +1,92 @@
+#include "ostree_packman_wrapper.h"
```

Comment:
Not a big deal, but it seems like unnecessary deviation to use "packman" here and "pacman" everywhere else.